.github/workflows/vulnerability-scanner-generate-database.yml

```diff
@@ -0,0 +1,99 @@
+name: Vulnerability Scanner - Generate database
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+# Only for test. delete pull_request: in release
+  pull_request:
+    # Pull request events
+    types: [ synchronize, opened, reopened, ready_for_review ]
+    # Path filtering
+    paths:
+      - ".github/workflows/vulnerability-scanner-generate-database.yml"
+      - "src/shared_modules/router/**"
+      - "src/shared_modules/content_manager/**"
+      - "src/shared_modules/indexer_connector/**"
+      - "src/wazuh_modules/vulnerability_scanner/**"
+      - "src/wazuh_modules/http-request/**"
+
+jobs:
+  vulnerability-scanner-updaload-database:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Project dependencies
+        uses: ./.github/actions/vulnerability_scanner_deps
+
+      ########################
+      # Compilation          #
+      ########################
+
+      # Router
+      - name: Router - Compilation and test
+        uses: ./.github/actions/compile_and_test
```

Comment:
We should avoid testing each module and only compile it.

Reply:
For the moment it will be handled like this.